src/serializers/stringSerializer.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  deserialize: string => string,
+  serialize: val => {
+    try {
```

Comment:
The store prevents `null` or `undefined` values from reaching the serializer:

https://github.com/jtompkins/rehash/blob/master/src/store.js#L104

It's probably still a good idea to handle those properly though.

Reply:
Ooooh, nice, validation :D If that's the case, I'd prefer to remove my try/catch... `{}` becomes `[object Object]` when turned into a string, and `[]` becomes `''` (empty string). Does that seem ok to you, or should we still return ''?